public_api/ddtrace/settings/http.pyi

```diff
@@ -0,0 +1,11 @@
+import Any
+
+log: Any
```

Comment:
probably worth adding something to doxxie to filter out top level `log`/`logger` instances